ui/guest/src/classes/ContentController.ts

```diff
@@ -958,19 +961,21 @@ function reducer(lookupTable: LookupTable<ContentInstance>, model: ContentInstan
 
 }
 
-function getResult(model: ContentInstance, fieldId: string, index: string | number): number[] | string[] {
-  const isStringIndex = typeof index === 'string';
+//function getResult(model: ContentInstance, fieldId: string, index: string | number): number[] | string[] {
+function getResult(collection: string[], index): string[] {
   const parsedIndex = parseInt(popPiece(`${index}`), 10);
-
-  const collection = isStringIndex
-    ? ModelHelper.extractCollection(model, fieldId, index)
-    : ModelHelper.value(model, fieldId);
-
```

Comment:
This looks dangerous. Sure? Need to test all kinds of sort/insert

Reply:
yes its works very well